content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -269,3 +269,5 @@ To understand the context for why Kubernetes wraps a common Pod API in other res
   * [Marathon](https://mesosphere.github.io/marathon/docs/rest-api.html)
   * [Omega](https://research.google/pubs/pub41684/)
   * [Tupperware](https://engineering.fb.com/data-center-engineering/tupperware/).
+
+See {{< api-reference page="workloads/pod-v1" >}}.
```

Comment:
nit: Is this added as a list item?

Reply:
This is an example to demonstrate the use of the api-reference shortcode, I think there is more work to decide where to place these links.

Reply:
OK. So, you could either remove it or find the right place to use the shortcode.

Reply:
I guess you have other ideas for where to use this shortcode.

Reply:
Yes, I have some ideas and have to work on this